### PR TITLE
Created a base pages setup

### PR DESCRIPTION
**Pull request by RJaliliSTFC**
Creating a foundation to work on
Imported old images and plain text

### CODE REVIEW

File: Templates/sla.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% extends "base.html" %}
+{% block title %}SLA{% endblock %}
+{% block content %}
+<p>The SCD Cloud is a service developed for use within Scientific Computing and the wider STFC for developing and supporting projects. SCD Cloud usage must comply with the <a href="https://cloud.stfc.ac.uk/tos">Terms of Service</a>. If you have any questions regarding the following, please contact <a href="mailto:cloud-support@gridpp.rl.ac.uk">cloud-support@gridpp.rl.ac.uk</a></p>
```
Comment (meoflynn) [inline Templates/sla.html]: Just need to update the cloud support email to `cloud-support@stfc.ac.uk`

File: Templates/tos.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <li>You UNDERSTAND that, whilst best effort is made to provide a stable platform, VMs may be subject to interruption and/or data loss at any time without notice (this is a development platform and should not be used for production services)</li>
+    <li>With respect to any SOFTWARE that you install you MUST ensure that all applicable license and terms and conditions of use are met </li>
+    <li>You MUST NOT use the service in any way related to providing any commercial service or running a private business</li>
+    <li>You MUST report any suspected or actual security incident or other misuse of the VM immediately to <a href="mailto:cloud-support@helpdesk.gridpp.rl.ac.uk">cloud-support@helpdesk.gridpp.rl.ac.uk</a> and notify the appropriate STFC security contact by following the procedure at <a href="https://staff.stfc.ac.uk/core/security/information/Pages/Incidents.aspx">https://staff.stfc.ac.uk/core/security/information/Pages/Incidents.aspx</a></li>
```
Comment (meoflynn) [inline Templates/tos.html]: Same as above (`cloud-support@stfc.ac.uk`)

File: cloud.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+from flask import Flask, render_template, request, url_for, redirect, session
+
+app = Flask(__name__)
+app.secret_key = b'r4nd0ms3cr3tk3y'
```
Comment (meoflynn) [inline cloud.py]: Please replace it with a placeholder to make it clearer that it's just a placeholder for a key

File: templates/sla.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
+<p>The SCD Cloud is a service developed for use within Scientific Computing and the wider STFC for developing and supporting projects. SCD Cloud usage must comply with the <a href="https://cloud.stfc.ac.uk/tos">Terms of Service</a>. If you have any questions regarding the following, please contact <a href="mailto:cloud-support@gridpp.rl.ac.uk">cloud-support@gridpp.rl.ac.uk</a></p>
```
Comment (meoflynn) [inline templates/sla.html]: Just need to update the email address here too (`cloud-support@stfc.ac.uk`)

File: templates/tos.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    <li>You UNDERSTAND that, whilst best effort is made to provide a stable platform, VMs may be subject to interruption and/or data loss at any time without notice (this is a development platform and should not be used for production services)</li>
+    <li>With respect to any SOFTWARE that you install you MUST ensure that all applicable license and terms and conditions of use are met </li>
+    <li>You MUST NOT use the service in any way related to providing any commercial service or running a private business</li>
+    <li>You MUST report any suspected or actual security incident or other misuse of the VM immediately to <a href="mailto:cloud-support@helpdesk.gridpp.rl.ac.uk">cloud-support@helpdesk.gridpp.rl.ac.uk</a> and notify the appropriate STFC security contact by following the procedure at <a href="https://staff.stfc.ac.uk/core/security/information/Pages/Incidents.aspx">https://staff.stfc.ac.uk/core/security/information/Pages/Incidents.aspx</a></li>
```
Comment (meoflynn) [inline templates/tos.html]: Same as above

**Review by meoflynn — CHANGES_REQUESTED** [review]
Looks good to me, just some minor changes needed to update the email addresses in the html files

File: cloud.py [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+#         return render_template("loginpage.html")
+
+# @app.route("/user")
+# def user():
+#     if "user" in session:
+#         user = session["user"]
+#         return render_template("testpage.html", passdata=user)
+#     else:
+#         return redirect(url_for("login"))
+
```
Comment (meoflynn) [inline cloud.py:38]: Has this page been removed or is it to be added in a future PR and just commented out for now?

Comment (RJaliliSTFC) [reply cloud.py:38]: For future PR and commented out for now

**Review by meoflynn — APPROVED** [review]

**Pull request merged**